### PR TITLE
Acb/version0.8

### PR DESCRIPTION
**Pull request by alexcb**
VERSION 0.8 is equivalent to

    VERSION \
    --no-network \
    --arg-scope-and-set \
    --use-docker-ignore \
    --pass-args \
    --global-cache \
    --cache-persist-option \
    --git-refs \
    --use-visited-upfront-hash-collection \
    --use-function-keyword \
    0.7

### CODE REVIEW

**Review by vladaionescu — COMMENTED** [review]
Should we create a `main-0.7` branch, just in case we still need to do a 0.7 release before 0.8 lands?

**Comment by alexcb** [comment]
> Should we create a `main-0.7` branch, just in case we still need to do a 0.7 release before 0.8 lands?

That sounds reasonable, branches are easy enough to create/delete.

**Review by idodod — APPROVED** [review]
`locally-in-function` is mostly a copy of `locally-in-command` where `FUNCTION` is used instead of `COMMAND`,
so I think we should be able to remove `locally-in-command`

**Review by vladaionescu — APPROVED** [review]

File: tests/Earthfile [hunk 1]
```diff
@@ -951,20 +933,12 @@ save-artifact-selective-referencing-remote:
...
-    DO +RUN_EARTHLY --target=github.com/earthly/test-remote/output/v0.5:main+savelocal
-    # if it were to be saved, it should show up under output/testfile; but we're going to check for
-    # a) no output directory (which also means output/testfile doesn't exist), and
-    # b) that ./testfile wasn't accidentally created.
-    RUN test ! -d output && \
-        test ! -f testfile
-
 save-remote-artifact-selective:
-    DO +RUN_EARTHLY --target=github.com/earthly/test-remote/output/versioned:main+savelocal
+    DO +RUN_EARTHLY --target=github.com/earthly/test-remote/output/versioned:acb/v0.8+savelocal
```
[the remark this answers is not in the document]
Comment (alexcb) [reply tests/Earthfile:937]: thanks for the reminder :)

**Pull request merged**